### ISSUE TITLE
Edit file only on remote without local download

### ISSUE THREAD

**Issue by IronMyke** [icetee/remote-ftp#265]
Is it possible to edit and save a file without downloading it? (maybe downloading it to a hidden temp folder, uploading when saving, and deleting when closing?)

**Comment by newelement** [comment]
Yeah, this extension will be complete for me when this happens. Many cases where wp-config.php needs to only be changed on the server side for many reasons. If it overwrites my local file I lose my DB connection info etc...

**Comment by brianjcohen** [comment]
+1 on this. At the very least it would be nice to avoid having a trail of remote edited files accumulating locally.

**Comment by jpxd** [comment]
This is the default Remote-FTP workflow (to download everything in the local project)... 
But someone could implement a right click option like "edit only on remote" for files in the remote tree view which uses a tmp file on the local end.

**Comment by brianjcohen** [comment]
@jpxd I would take it a step further and make it either a global or per-project setting to allow only temp local files to be used (edit only on remote), rather than forcing the user to always make multiple clicks as you described if that user wanted it to never persist locally.

**Comment by kartagis** [comment]
What a wonderful idea! I've been thinking about the same thing myself. This would allow us to keep our local project folder non-cluttered.
